### ISSUE TITLE
How can I get hyperparameter in ScriptMoudle using libtorch?

### ISSUE THREAD

**Issue by yy738686337** [pytorch/pytorch#29808]
After I trained a model in python and save it with `torch.jit.trace,` how can I get the parameter e.g. 
stride or padding size of `Conv2d `when loading the model in libtorch?
by the way, the model is created by` torchvision.models`.

cc @suo

**Comment by driazati** [comment]
You can grab parameters off a `script::Module` with [`script::Module::attr`](https://pytorch.org/cppdocs/api/structtorch_1_1jit_1_1script_1_1_module.html#exhale-struct-structtorch-1-1jit-1-1script-1-1-module). You can also print the whole model with `script::Module::dump()` if that's what you're looking for. Closing since this is a help question (please put any follow ups on the forums [here](https://discuss.pytorch.org/c/jit))
